usvg/src/convert/filter.rs

```diff
@@ -490,10 +490,7 @@ fn convert_fe_convolve_matrix(
         _      => tree::FeEdgeMode::Duplicate,
     };
 
-    let preserve_alpha = match fe.attribute(AId::PreserveAlpha).unwrap_or("false") {
-        "true" => true,
-        _      => false,
-    };
+    let preserve_alpha = matches!(fe.attribute(AId::PreserveAlpha).unwrap_or("false"), "true");
```

Comment:
Can't we use just `fe.attribute(AId::PreserveAlpha).unwrap_or("false") == "true"` here?